src/app/gigs/gigs-comms.tsx

```diff
@@ -94,18 +96,26 @@ export const GigsComms = ({
 }: GigsCommsProps) => {
   const [activeTab, setActiveTab] = useState<ActiveTab>(ActiveTab.Chat);
   const [offers, setOffers] = useState([]);
+  const { toast } = useToast();
 
   const {
     isPending: getContractPending,
     data: contractData,
     // error: getContractError,
-    // isError: isGetContractError,
+    isError: isGetContractError,
   } = useGetContractForOffer({
     currentOfferId,
     isOpen,
     activeTab,
   });
 
+  useEffect(() => {
+    toast({
+      title: 'Unable to fetch contract information',
+      description: 'If the problem persists, please contract support',
```

Comment:
```suggestion
      description: 'If the problem persists, please contact support',
```

When we ask people to contact support, there should be a link for them to do so. Maybe we should chat with Rami about this?